phones/models.py

```diff
@@ -1,4 +1,241 @@
+from datetime import datetime, timedelta
+import math
+import random
+
+from django.apps import apps
+from django.contrib.auth.models import User
+from django.conf import settings
+from django.core.exceptions import BadRequest
+from django.db.migrations.recorder import MigrationRecorder
 from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch.dispatcher import receiver
+from django.urls import reverse
+
+
+MAX_MINUTES_TO_VERIFY_REAL_PHONE = 5
+
+
+def verification_code_default():
+    return str(math.floor(random.random()*999999)).zfill(6)
+
+
+def verification_sent_date_detaul():
```

Comment:
Suggest changing the name:

```suggestion
def verification_sent_date_default():
```

This will have to be reflected several other places, like the migration.